.github/workflows/jest.yml

```diff
@@ -1,58 +1,130 @@
-name: Run test and report code coverage on /react
+name: Run test and report code coverage on /react and /backend.ai-ui
 
 on:
   pull_request:
-    branches: [ "main" ]
     paths:
       - react/src/**
       - react/package.json
+      - packages/backend.ai-ui/src/**
+      - packages/backend.ai-ui/package.json
+
+permissions:
+  contents: read
```

Comment:
```suggestion
  contents: read
  pull-requests: write
```

Reply:
Since the default permission for that workflow is `content: read`, and the react-coverage and backend-ai-ui-coverage tests have separate `pull-request: write` permissions, I'll merge without modification. @agatha197